lib/inky/inkyio.ex

```diff
@@ -71,7 +71,7 @@ defmodule Inky.InkyIO do
   # SPI data
 
   defp send_data(pins, data) when is_integer(data) do
-    spi_write(pins, @spi_data, <<data>>)
+    spi_write(pins, @spi_data, <<data::unsigned-little-integer-16>>)
```

Comment:
Are we just being extra-explicit about the format here? It is probably the third time we are enforcing it, but I guess that might be for the best.

Reply:
Good point. I'll remove this.

Reply:
Actually, it is _NOT_ being enforced for anything _BUT_ the packed numbers... but since it was already working, there's no point in doing this.... so I'll still remove it :P